docs/requirements-docs.in

```diff
@@ -13,5 +13,6 @@ docutils~=0.16
 PyYAML==6.0.2
 urllib3~=1.26.19
 wheel==0.44.0
-typing-extensions==4.12.2
+typing-extensions>=4.11
```

Comment:
Curious why this was lowered?

Reply:
this has no impact on the lib and is strictly for doc generation.  Opening the dependencies ease lib conflicts